qutebrowser/config/websettings.py

```diff
@@ -247,6 +247,7 @@ def clear_private_data() -> None:
     if objects.backend == usertypes.Backend.QtWebEngine:
         from qutebrowser.browser.webengine import webenginesettings
         webenginesettings.init_private_profile()
+        webenginesettings.global_settings.init_settings()
```

Comment:
If we have an `webenginesettings.init_profile()` to initialize a new profile, I suppose this could be called there (instead of being called here and separately in `webenginesettings.init()`)